test/integration/dt2js_integration.test.js

```diff
@@ -135,44 +135,33 @@ types:
   })
 })
 
-describe('dt2js function integration test with --validation option', function () {
+describe('dt2js function integration test with --validate option', function () {
   const dt2js = dt2jsMod.__get__('dt2js')
   const data = `
 #%RAML 1.0 Library
 
 types:
   PersonAge:
-    type: number
-    minimum: 1
-    maximum: 50
-    format: int32`
-  let revert
-  beforeEach(function () {
-    revert = dt2jsMod.__set__({
-      migrateDraft: function (name) {
-        return {
-          $schema: 'http://json-schema.org/draft-04/schema',
-          required: 'asdasdasd'
-        }
-      }
-    })
-  })
-  afterEach(function () { revert() })
-  context('when --validation option is passed', function () {
+    type: string
+    enum: 4
+  PersonName:
+    type: string
+`
+  context('when --validate option is passed', function () {
     it('should validate output json schema', async function () {
       try {
         await dt2js(data, 'PersonAge', { validate: true })
         throw new Error('Expected to fail')
       } catch (e) {
         expect(e.message).to.equal(
-          'Invalid JSON Schema: data.required should be array')
+          "schema is invalid: data.definitions['PersonAge'].enum should NOT have less than 1 items")
```

Comment:
is this the right error message? to me, it seems like the issue is that it's not an array rather than the number of items.

Reply:
It's an validation error message returned by Ajv so it's their wording, not ours.
I've updated this test to display the message we should return during validation.
Old message (`schema is invalid`) is thrown when migrating draft with `json-schema-migrate` because it also uses Ajv (older version) to validate input json schema.